docs/REST API/rest-api.md

```diff
@@ -9709,6 +9711,7 @@ Update a package to install on macOS, Windows, or Linux (Ubuntu) hosts.
 | id | integer | path | ID of the software title being updated. |
 | software        | file    | form | Installer package file. Supported packages are .pkg, .msi, .exe, .deb, and .rpm.   |
 | team_id         | integer | form | **Required**. The team ID. Updates a software package in the specified team. |
+| categories        | string | form | One or more (comma-separated) category name, as shown in the UI. Sets the software package to the specified categories. |
```

Comment:
@eugkuo here and for upload package could we change the type to be `array` and remove the bit about the comma-separated list? This will make it consistent with how we implement labels (and make it a bit easier to parse the response in the server code).

Reply:
@jahzielv I was eng DRI on this I think, so answer's yes.

Reply:
@iansltx cool! I can push that change if no objection